### PR TITLE
Added azure-storage plugin into the plugin marketplace

### PR DESCRIPTION
**Pull request by deepan10**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by tudi2d — CHANGES_REQUESTED** [review]
Thanks for adding this plugin! Needs some clarification & updated documentation :)

File: microsite/data/plugins/azure-storage.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Azure Storage Explorer
+author: Deepankumar
+authorUrl: https://github.com/deepan10
+category: Infrastructure
+description: Explore Azure Storage Blobs in Backstage.
+documentation: https://github.com/deepan10/backstage-plugins/tree/main/plugins/azure-storage
```
[the remark this answers is not in the document]
Comment (deepan10) [reply microsite/data/plugins/azure-storage.yaml:7]: Updated the single page documentation for both Frontend and backend plugin setup along with the screenshots.

**Comment by deepan10** [comment]
@tudi2d  Prettier is successful in my local. it is failing in the pipeline can you re-trigger its once.

image

**Comment by freben** [comment]
@deepan10 did you run it inside the microsite folder? That check is not likely to be flaky and have a different outcome next time.

**Comment by deepan10** [comment]
Thanks @freben It worked! 😄

File: microsite/data/plugins/azure-storage.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tudi2d) [reply microsite/data/plugins/azure-storage.yaml:7]: Awesome - this is great!

**Review by tudi2d — APPROVED** [review]

**Comment by deepan10** [comment]
Hello @freben / @tudi2d  / @Rugvip,

Can you please re-open this PR #21675 .

**Comment by freben** [comment]
Sure.

**Review by freben — APPROVED** [review]

**Pull request merged**